test/run-clang-tidy.sh

```
@@ -0,0 +1,23 @@
+#!/bin/bash
+#
+# Execute from project root directory
+#
+
+set -e
+
+export CMAKE_EXPORT_COMPILE_COMMANDS=ON
+export CMAKE_BUILD_PARALLEL_LEVEL=$(nproc)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)